.eslintrc.js

```diff
@@ -22,6 +22,12 @@ module.exports = {
     jest: true,
     node: true,
   },
+  overrides: [{
+    files: ["**/*.tsx"],
+    rules: {
+        "react/prop-types": "off"
```

Comment:
This is not needed because typescript itself already forces the type declaration of props.